### PR TITLE
purpose place in complementPostObject

### PR DESCRIPTION
**Pull request by NiclasNorin**

### CODE REVIEW

File: library/Helper/Location.php [hunk 1]
```diff
@@ -51,25 +51,33 @@ public static function createPins(array $posts, bool $displayGoogleMapsLink): ar
      * @param boolean $displayGoogleMapsLink Whether to include a Google Maps link in the tooltip.
      * @return array $pin The pin data.
      */
-    public static function createPin($post, $displayGoogleMapsLink = false): array
+    public static function createPin($post): array
```
Comment (annalinneajohansson) [inline library/Helper/Location.php:54]: Not sure we **always** want to show the directions-link to Google Maps?

File: library/Helper/Location.php [hunk 2]
```diff
@@ -51,25 +51,33 @@ public static function createPins(array $posts, bool $displayGoogleMapsLink): ar
...
+                    'excerpt' => $post->post_excerpt ?? '',
                     'url' => $permalink,
                 ],
+                /* Maybe not needed? */
                 'url' => $permalink,
             ];
+            
             // Add directions link to tooltip
-            if ($displayGoogleMapsLink) {
+            if (!empty($post->location['lat']) && !empty($post->location['lng'])) {
```
Comment (annalinneajohansson) [inline library/Helper/Location.php:74]: You don't need to check with empty() here, you already do so on line 57.

File: library/Helper/Location.php [hunk 3]
```diff
@@ -51,25 +51,33 @@ public static function createPins(array $posts, bool $displayGoogleMapsLink): ar
...
+                $directions = 'https://www.google.com/maps/dir/?api=1&destination=' . $post->location['lat'] . ',' . $post->location['lng'] . '&travelmode=transit';
+            }
+            
+            if ($directions) {
```
Comment (annalinneajohansson) [inline library/Helper/Location.php:78]: See https://github.com/helsingborg-stad/Municipio/pull/624/files#r1229423504

File: library/Helper/Location.php (hunk 1 above)
Comment (NiclasNorin) [reply library/Helper/Location.php]: This will only add the correct data to every post object, the view will handle the "show condition"

**Pull request merged**